tests/src/test/resources/org/jboss/forge/grammar/java/MockAnnotatedEnumConstant.java

```diff
@@ -6,11 +6,11 @@
  */
 package org.jboss.forge.grammar.java;
 
-import org.jboss.forge.test.grammar.java.common.MockAnnotation;
-import org.jboss.forge.test.grammar.java.common.MockNestingAnnotation.MockNestedAnnotation;
-import org.jboss.forge.test.grammar.java.common.MockNestingAnnotation;
+import org.jboss.forge.test.roaster.model.common.MockAnnotation;
```

Comment:
I am a bit unsure whether this should be changed. I don't quite remember why we have a grammar package, but I'd rather keep it unchanged for now

Reply:
The problem is, that we already set Annotation needs to be removed and this annotation is not available as a class object, so a removeImport call is not possible. That was the reason I changed it